synapse/cortex.py

```diff
@@ -3198,6 +3202,25 @@ async def itemsStormVar(self):
     async def _cortexHealth(self, health):
         health.update('cortex', 'nominal')
 
+    async def _migrateTaxonomyIface(self):
+
+        extforms = await (await self.hive.open(('cortex', 'model', 'forms'))).dict()
+
+        for formname, basetype, typeopts, typeinfo in extforms.values():
+            try:
+                ifaces = typeinfo.get('interfaces')
+
+                if ifaces and 'taxonomy' in ifaces:
```

Comment:
Lets log the value that is being migrated so that it is clear what has changed

Reply:
added a warning level log message